cmd/grr/workflow.go

```diff
@@ -218,12 +221,10 @@ func providersCmd(registry grizzly.Registry) *cli.Command {
 	return cmd
 }
 
-func grizzlyOptsFromCmd(cmd *cli.Command) grizzly.GrizzlyOpts {
-	return grizzly.GrizzlyOpts{
-		ConfigFile:   cmd.Flags().StringP("config", "c", "", "config file"),
-		Targets:      cmd.Flags().StringSliceP("target", "t", nil, "resources to target"),
-		JsonnetPaths: cmd.Flags().StringSliceP("jpath", "J", getDefaultJsonnetFolders(), "Specify an additional library search dir (right-most wins)"),
-	}
+func defaultGrizzlyFlags(opts *grizzly.GrizzlyOpts, fs *pflag.FlagSet) {
```

Comment:
What about:

```go
func defaultGrizzlyFlags(fs *pflag.FlagSet) grizzly.GrizzlyOpts {
    var opts grizzly.GrizzlyOpts
    // ...
    return opts
}
```

Reply:
Only if you can explain why that didn't work when I tried it.